nevergrad/functions/images/core.py

```diff
@@ -6,8 +6,11 @@
 from pathlib import Path
 
 import numpy as np
-import nevergrad as ng
 import PIL.Image
+import torch.nn as nn
+import torch
+
+import nevergrad as ng
```

Comment:
importing Nevergrad inside Nevergrad ? you just have to import parameters as p, right ?

Reply:
I like it better that way actually, so that it's closer to a user's code.
I do this mostly for parameters

Reply:
That's jeremy's line of code  so I tend to agree with him ^^